vendor/bat-native-ledger/src/bat/ledger/internal/media/reddit.cc

```diff
@@ -237,30 +237,27 @@ void Reddit::OnUserPage(
   const std::string publisher_name = GetPublisherName(response);
   const std::string user_id = GetUserId(response);
   const std::string publisher_key = GetPublisherKey(user_id);
-  const std::string media_key = GetMediaKey(user_name, REDDIT_MEDIA_TYPE);
+  const std::string media_key =
+      braveledger_media::GetMediaKey(user_name, REDDIT_MEDIA_TYPE);
 
   ledger::VisitData new_visit_data(visit_data);
   new_visit_data.provider = REDDIT_MEDIA_TYPE;
   new_visit_data.url = GetProfileUrl(user_name);
   new_visit_data.favicon_url = GetProfileImageUrl(response);
   new_visit_data.name = publisher_name.empty() ? user_name : publisher_name;
-
-  ledger_->SaveMediaVisit(
-    publisher_key,
-    new_visit_data,
-    0,
-    window_id,
-    std::bind(&Reddit::OnUserActivity,
```

Comment:
why do we want to bind to `OnUserActivity` when we save data?

Reply:
This function is related to panel

Reply:
`OnPanelPublisherInfo` is already handled in `SaveMediaVisit`

Reply:
so when you save panel will be updated for you. This callback is added for a different thing

Reply:
removed